modulefiles/workflow_utils.hera

```diff
@@ -20,7 +20,9 @@ module load sp/2.3.3
 module load ip/3.3.3
 module load nemsio/2.5.2
 module load nemsiogfs/2.5.3
-#module load ncio/1.0.0
+module load ncio/1.0.0
```

Comment:
Done

Reply:
I think w3emc/2.7.3 depends on netcdf.
Later versions of w3emc removed that dependency.  These should be moved before the nceplibs list start since these are more closer to third-party than the nceplibs ones.